### ISSUE TITLE
Question about masks

### ISSUE THREAD

**Issue by ctxya1207** [wangbohan97/Kolors-MPS#14]
mask = torch.where(sim_text_condition > 0.01, 0, float('-inf')) # B*1*77，How is the 0.01 here designed?
